taskcluster/android_taskgraph/transforms/gradle_optimization.py

```diff
@@ -55,6 +55,7 @@ def extend_optimization_if_one_already_exists(config, tasks):
         optimization = task.get("optimization")
         if optimization:
             skip_unless_changed = optimization["skip-unless-changed"]
+            skip_unless_changed.append(f"{config.path}/**")
```

Comment:
IIUC, this path just takes the kinds into account. In which case, let's not optimize them out unless there's a change in their kind folder

```suggestion
            skip_unless_changed.append(f"{config.path}/{task.kind}/**")
```

Reply:
config.path already includes the kind, it's `taskcluster/ci/{kind}` typically.

Reply:
Oh great! :shipit: then